plugin.js

```diff
@@ -563,92 +592,41 @@ function makeWorklet(t, fun, state) {
         closureGenerator.generate(t, variables, closure.keys())
       )
     ),
-    t.expressionStatement(
-      t.assignmentExpression(
-        '=',
-        t.memberExpression(privateFunctionId, t.identifier('asString'), false),
-        t.stringLiteral(funString)
-      )
-    ),
     t.expressionStatement(
       t.assignmentExpression(
         '=',
         t.memberExpression(
           privateFunctionId,
-          t.identifier('__workletHash'),
+          t.identifier('__initData'),
           false
         ),
-        t.numericLiteral(workletHash)
+        initDataId
       )
     ),
     t.expressionStatement(
       t.assignmentExpression(
         '=',
         t.memberExpression(
           privateFunctionId,
-          t.identifier('__location'),
+          t.identifier('__workletHash'),
           false
         ),
-        t.stringLiteral(location)
+        t.numericLiteral(workletHash)
       )
     ),
   ];
 
-  if (sourceMapString) {
-    statements.push(
+  if (!isRelease()) {
+    statements.unshift(
       t.expressionStatement(
         t.assignmentExpression(
           '=',
-          t.memberExpression(
-            privateFunctionId,
-            t.identifier('__sourceMap'),
-            false
-          ),
-          t.stringLiteral(sourceMapString)
-        )
-      )
-    );
-  }
-
-  if (!isRelease()) {
-    statements.unshift(
-      t.variableDeclaration('const', [
-        t.variableDeclarator(
-          t.identifier('_e'),
+          t.memberExpression(initDataId, t.identifier('stackDetails'), false),
           t.arrayExpression([
             t.newExpression(t.identifier('Error'), []),
             t.numericLiteral(lineOffset),
             t.numericLiteral(-20), // the placement of opening bracket after Exception in line that defined '_e' variable
           ])
-        ),
-      ])
-    );
-    statements.push(
-      t.expressionStatement(
-        t.assignmentExpression(
-          '=',
-          t.memberExpression(
-            privateFunctionId,
-            t.identifier('__stackDetails'),
-            false
-          ),
-          t.identifier('_e')
-        )
-      )
-    );
-  }
-
-  if (options && options.optFlags) {
-    statements.push(
-      t.expressionStatement(
-        t.assignmentExpression(
-          '=',
-          t.memberExpression(
-            privateFunctionId,
-            t.identifier('__optimalization'),
```

Comment:
Since we no longer read `options.optFlags`, I think we can also remove the assignment `options.optFlags = isPossibleOptimization(transformed.ast);` along with the definition of `isPossibleOptimization`.